src/main/java/com/codurance/retropolis/repositories/CardRepository.java

```diff
@@ -7,4 +7,6 @@
   Card insert(Card newCard);
 
   void delete(Long cardId);
+
+    Card update(Long cardId, String newText);
```

Comment:
maybe rename this too, to updateText. like with the controller function

Reply:
Resolved on https://github.com/codurance/Retropolis-BE/pull/26/commits/7f3db9a51a8e925f4a9b046e6b1fc724e4fa5702